modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/function/CSSFunction.java

```diff
@@ -60,19 +60,16 @@ public CSSFunction() {
     }
 
     @Override
-    public <T> T evaluate(Object object, Class<T> context) {
+    public Object evaluate(Object object) {
```

Comment:
Is there any merit to instead just adding:
```
public Object evaluate(Object object) {
   return evaluate(Object object, null);
}
```
in order to preserve the generic implementation?

Reply:
Not for this one I believe... these functions were implemented "upside down", the base class is designed so that one implements the version of evaluate without the context, and the base class takes care of using converters if the version with the context is called. Now I understand if the implementation can do something intelligent with the context, then both need to be implemented, but this particular function has no use for it.